src/Resources/config/cache_manager_commands.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+
+        <service id="fos_http_cache.command.invalidate_path" class="FOS\HttpCacheBundle\Command\InvalidatePathCommand">
+            <argument type="service" id="fos_http_cache.cache_manager" />
+            <tag name="console.command" command="fos:httpcache:invalidate:path"/>
+        </service>
+
+        <service id="fos_http_cache.command.invalidate_regex" class="FOS\HttpCacheBundle\Command\InvalidateRegexCommand">
+            <argument type="service" id="fos_http_cache.cache_manager" />
+            <tag name="console.command" command="fos:httpcache:invalidate:regex"/>
+        </service>
+
+        <service id="fos_http_cache.command.refresh_path" class="FOS\HttpCacheBundle\Command\RefreshPathCommand">
+            <argument type="service" id="fos_http_cache.cache_manager" />
+            <tag name="console.command"/>
```

Comment:
this is missing the "command" attribute compared to the other command. but since you set the static property now, you don't need to specify the command name in the services anymore